internal/compare/compare_test.go

```diff
@@ -215,3 +215,33 @@ func TestServicesError(t *testing.T) {
 		})
 	}
 }
+
+func TestRelativePath(t *testing.T) {
+	t.Parallel()
+	tests := []struct {
+		desc string
+		path string
+		want string
+	}{
+		{
+			desc: "good case",
+			path: "/home/foo/bar/buz/buz.go",
+			want: "buz/buz.go",
+		},
+		{
+			desc: "fallback case where we just return file name",
+			path: "foo/buz.go",
+			want: "buz.go",
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.desc, func(t *testing.T) {
```

Comment:
Need `tt := tt` here or we'll run into the closure scoping issue.
```suggestion
	for _, tt := range tests {
		tt := tt
		t.Run(tt.desc, func(t *testing.T) {
```